io.github.slgobinath.SafeEyes.yaml

```diff
@@ -0,0 +1,95 @@
+id: io.github.slgobinath.SafeEyes
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+command: safeeyes
+
+finish-args:
+  - "--share=network"
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--system-talk-name=org.freedesktop.StatusNotifierItem.Menu" # for tray icon
+  - "--talk-name=org.mpris.MediaPlayer2.*" # to be able to pause media player on break
+  # various DE tray support
+  - "--talk-name=com.canonical.indicator.application"
+  - "--talk-name=com.canonical.AppMenu.Registrar"
+  - "--talk-name=org.ayatana.indicator.application"
+  - "--talk-name=org.gnome.Mutter.IdleMonitor"
+  - "--talk-name=org.kde.StatusNotifierWatcher"
+  - "--env=GTK_PATH=/app/lib/gtk-3.0"
+
+cleanup: # Cleanup after the python modules
+  - /cache
+  - /man
+  - /share/aclocal
+  - /share/devhelp
+  - /include
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - /share/man
+  - /lib/*.a
+  - /lib/*.la
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-introspection-12.10.json # app indicator support
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - shared-modules/libcanberra/libcanberra.json
+
+  # auto generated pip dependencies for safeeyes
+  # install pycairo first or build fails
+  # install order is: pycairo -> setuptools-scm -> safeeyes
+  # `python3 flatpak-pip-generator.py  --runtime='org.freedesktop.Sdk//22.08' PKGNAME`
+  - pypi-dependencies.json
+
+  - name: dbus-python # dbus - for tray
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: "https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.3.2.tar.gz"
+        sha256: ad67819308618b5069537be237f8e68ca1c7fcc95ee4a121fe6845b1418248f8
+
+  - name: wlrctl # required for 'wayland do not disturb' plugin
```

Comment:
why do you need this if you don't have the wayland socket.

Reply:
Even though the app only runs through x11, it can still detect wayland and `wlrctl` is required to detect full screen

Reply:
how does it detect wayland?

Reply:
with [this function](https://github.com/slgobinath/SafeEyes/blob/cae19b0f28f14036d1a2a4c840bf3e6f68c02b6f/safeeyes/utility.py#L299) 

without wlrctl you can see [the warning](https://github.com/slgobinath/SafeEyes/blob/cae19b0f28f14036d1a2a4c840bf3e6f68c02b6f/safeeyes/plugins/donotdisturb/plugin.py#L52)

Reply:
do you test is inside the flatpak sandbox? Because this detection is broken.

Reply:
Yeah, you're right. I'm going to keep to fiddle with it. It probably acts like that because of missing wayland socket